internal/services/containers/kubernetes_cluster_addons_resource_test.go

```diff
@@ -135,47 +117,28 @@ func TestAccKubernetesCluster_addonProfileOMSToggle(t *testing.T) {
 			Config: r.addonProfileOMSConfig(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That(data.ResourceName).Key("addon_profile.0.http_application_routing.#").HasValue("0"),
-				check.That(data.ResourceName).Key("default_node_pool.0.node_count").HasValue("1"),
-				check.That(data.ResourceName).Key("addon_profile.0.oms_agent.#").HasValue("1"),
-				check.That(data.ResourceName).Key("addon_profile.0.oms_agent.0.enabled").HasValue("true"),
-				check.That(data.ResourceName).Key("addon_profile.0.oms_agent.0.log_analytics_workspace_id").Exists(),
 			),
 		},
 		data.ImportStep(),
 		{
 			Config: r.addonProfileOMSDisabledConfig(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That(data.ResourceName).Key("addon_profile.0.http_application_routing.#").HasValue("0"),
-				check.That(data.ResourceName).Key("default_node_pool.0.node_count").HasValue("1"),
-				check.That(data.ResourceName).Key("addon_profile.0.oms_agent.#").HasValue("1"),
-				check.That(data.ResourceName).Key("addon_profile.0.oms_agent.0.enabled").HasValue("false"),
-				check.That(data.ResourceName).Key("addon_profile.0.oms_agent.0.log_analytics_workspace_id").HasValue(""),
 			),
 		},
 		data.ImportStep(),
+		// TODO 3.0 - Remove this step since this config will be identical to the one provided in the step above
```

Comment:
pedantic - but we should split this into two different steps so we can remove the 2.x test in the future